src/tests/Microsoft.Diagnostics.NETCore.Client/Microsoft.Diagnostics.NETCore.Client.UnitTests.csproj

```diff
@@ -1,5 +1,41 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <Target Name="_generateDARCVersionCs"
+          Inputs="$(VersionsPropsPath)"
+          Outputs="$(IntermediateOutputPath)/DARCVersion.g.cs">
+    <PropertyGroup>
+      <DARCVersionFile>
+<![CDATA[
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+// THIS FILE IS AUTO-GENERATED DURING BUILD.  MODIFICATIONS _WILL_ BE OVERWRITTEN!
+
+namespace Microsoft.Diagnostics.NETCore.Client
+{
+    public partial class CommonHelper
+    {
+        public static readonly string CurrentDARCVersion = "$(MicrosoftNETCoreAppVersion)";
+    }
+}
+]]>
+      </DARCVersionFile>
+    </PropertyGroup>
+
+    <WriteLinesToFile File="$(IntermediateOutputPath)/DARCVersion.g.cs"
```

Comment:
You're using the path `$(IntermediateOutputPath)/DARCVersion.g.cs` a few times. Perhaps make it a property at the beginning of the proj with a chunky comment explaining the purpose of these targets?